### PR TITLE
Add example and test

### PR DESCRIPTION
**Pull request by azu**
Add [example tests](http://efcl.info/2015/07/29/example-test-on-npm/) and integrate Travis CI.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,34 @@
-.textlintrc
-gulpfile.js
 test.md
 webdb.yml
+
+# Logs
```
Comment (azu) [inline .gitignore:4]: from https://github.com/github/gitignore/blob/master/Node.gitignore

File: example/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "private": true,
+  "scripts": {
+    "textlint": "gulp textlint",
+    "test": "npm i && npm run textlint"
+  },
+  "author": "azu",
+  "license": "MIT",
+  "dependencies": {
+    "gulp": "^3.9.0",
+    "gulp-textlint": "file:..",
```
Comment (azu) [inline example/package.json:13]: install as local module for test

File: package.json [hunk 3]
```diff
@@ -13,6 +13,9 @@
     "url": "https://github.com/nakajmg/gulp-textlint/issues"
   },
   "homepage": "https://github.com/nakajmg/gulp-textlint",
+  "scripts": {
+    "test": "cd example && npm test"
```
Comment (azu) [inline package.json:16]: run example

**Comment by nakajmg** [comment]
LGTM :+1:

**Pull request merged**